package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "lfpwebapp",
+  "version": "0.0.1",
+  "description": "This is a skeleton for a node.js + express.ts app.",
+  "main": "dist/server.js",
+  "scripts": {
+    "start": "npm run build && node public/server.js",
+    "build": "touch public && rm -dr public && tsc && cp -r src/views src/assets public/",
+    "devstart": "nodemon --config nodemon.json",
+    "tslint": "./node_modules/.bin/tslint -c tslint.json -p tsconfig.json",
```

Comment:
Since you implemented linting it make sense to extend it by SCSS linter. An example here: https://github.com/alphagov/govuk-design-system/blob/master/package.json#L23